### PR TITLE
DEVELOPER-4771 - Downloads page content is inaccessible

### PR DESCRIPTION
**Pull request by Dantheman720**
https://issues.jboss.org/browse/DEVELOPER-4771

To test: Visit the /downloads page and make sure that all downloads are appearing as expected. Also, you'll want to visit the product download pages for each product and make sure that the downloads are appearing.

### CODE REVIEW

**Comment by Dantheman720** [comment]
@RubyGemTSL - This should fix the download issues by pointing everything to staging unless it is on the live site. Since all of the download references are relative and we don't have a download-manager url available per environment, I think this may be the best solution for now.

**Comment by RubyGemTSL** [comment]
@Dantheman720 - we should be using production Keycloak and Download Manager on PR's. See expected result on https://issues.jboss.org/browse/DEVELOPER-4771. Unless you've been told otherwise?

**Comment by Dantheman720** [comment]
Oh! I'll get that fixed.

**Comment by RubyGemTSL** [comment]
Thanks @Dantheman720 .

It should be:
PR's -> Production Keycloak and Download Manager
Staging -> Staging Keycloak and Staging Download Manager

File: typescript/rhdp-downloads/rhdp-downloads-app.ts [hunk 1]
```diff
@@ -78,6 +78,9 @@ class RHDPDownloadsApp extends HTMLElement {
     }
 
     setProductsDownloadData(url) {
+        if(window.location.origin.indexOf('developers.redhat.com') == -1){
```
Comment (LightGuard) [inline typescript/rhdp-downloads/rhdp-downloads-app.ts]: Is this following our whitespace rules for typescript?

**Comment by Dantheman720** [comment]
@RubyGemTSL - I changed the downloads to point to Production unless the URL is staging. Keycloak shouldn't be a problem since the login page redirects to the environment relative to download manager. Is developers.stage.redhat.com the only staging environment that needs to point to the staging download maanger?

**Comment by RubyGemTSL** [comment]
@Dantheman720 yep, that's correct! Thank you!! I'll keep an eye out for this once it's built.

**Comment by RubyGemTSL** [comment]
@Dantheman720 the site preview still looks like it's trying to load downloads using the relative urls? For example 

`https://developers-pr.stage.redhat.com/pr/2251/export/download-manager/rest/available/rhel%2ceap%2cdevstudio%2cfuse%2cdatagrid%2ceap%2cwebserver%2ccdk%2cdevsuite%2camq%2cbrms%2cbpmsuite%2cdatavirt%2cmobileplatform%2copenshift%2copenjdk%2cdotnet/?nv=1`

**Comment by RubyGemTSL** [comment]
was the change made in drupal so it has to synch will production?

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
Yeah /downloads is still not accessible after a retest

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
@LightGuard - Do I need to close this and open another for the sync with prod? This is still pulling the relative urls and not the absolute.

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by LightGuard** [comment]
@RubyGemTSL are we good to merge?

**Comment by RubyGemTSL** [comment]
@LightGuard no, just talking with Dan. He has more work to do on this

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
@Dantheman720 unfortunately in the staging environment looks like staging is pointing to production. 

All downloads are pointing to production except for:

JBoss Web Server Downloads
Red Hat Application Migration Toolkit - redirects back to /downloads

Red Hat Container Development Kit - oops page (https://developers.stage.redhat.com/download-manager/file/devsuite-2.2.0-GA-bundle-installer-mac.zip)

Red Hat Development Suite
- oops page (https://developers.stage.redhat.com/download-manager/file/devsuite-2.2.0-GA-bundle-installer-mac.zip).

**Pull request merged**